pages/pets/index.js

```diff
@@ -76,34 +119,67 @@ const StyledForm = styled.form`
   margin-top: 2rem;
   margin-bottom: 2rem;
 
-  & div {
+  & fieldset {
+    border: none;
     display: flex;
     flex-direction: column;
-    gap: 0.5rem;
-
-    & input,
-    select {
-      width: 100%;
-      padding: 1rem;
-      border-radius: 0.5rem;
-      border: 1px solid grey;
-
-      -webkit-appearance: none;
-      -moz-appearance: none;
-      appearance: none;
-    }
+    gap: 1rem;
+    padding-top: 1rem;
+  }
```

Comment:
As the fieldset gets more than one CSS declaration I create a separate styled component. And this way the reusability of your component is more flexible. :)